ospd_openvas/gpg_sha_verifier.py

```diff
@@ -44,13 +44,18 @@ def create_hash(file: Path) -> str:
         # therefore a collision is not the end of the world and sha1 is more
         # than sufficient
         hasher = hashlib.sha1()
-        with file.open(mode="rb") as f:
-            for hash_file_bytes in iter(lambda: f.read(1024), b""):
-                hasher.update(hash_file_bytes)
-        return hasher.hexdigest()
+        try:
+            with file.open(mode="rb") as f:
+                for hash_file_bytes in iter(lambda: f.read(1024), b""):
+                    hasher.update(hash_file_bytes)
+            return hasher.hexdigest()
+        except:
```

Comment:
Maybe just catch a specific base error here? A catchall can hide serious issues.

Reply:
I guess [OSError](https://docs.python.org/3/library/exceptions.html#OSError) would be best here